R/check_if_installed.R

```diff
@@ -128,3 +153,18 @@ print.check_if_installed <- function(x, ...) {
     print_color(paste0("- ", names(x)[!x], collapse = "\n"), "red")
   }
 }
+
+get_dep_version <- function(dep, pkg = utils::packageName()) {
```

Comment:
I suggest we call this `.get_dep_version()` given that it's an internal function.